docs/manual/helloworld-unity-localvideo.md

```diff
@@ -1,38 +1,124 @@
 
 - In the **Hierarchy** window, select **Create** > **Create Empty**.
 - In the **Inspector** window, rename the newly-created game object to something memorable like "LocalMediaPlayer".
-- Press the **Add Component** button at the bottom of the window, and select **MixedReality-WebRTC** > **LocalVideoSource**.
-- This component needs to know which peer connection to use. Once again, use the asset selection window to assign our peer connection to the **Peer Connection** property.
+- Press the **Add Component** button at the bottom of the window, and select **MixedReality-WebRTC** > **Webcam Source**.
+
+![Create a webcam source](helloworld-unity-8.png)
+
+The webcam source component contains several interesting properties, divided into 2 groups. We detail these properties below for reference, but for now we can leave all of them to their default values.
+
+> [!IMPORTANT]
+> When using Mixed Reality Capture (MRC) on HoloLens, the Unity project needs to be configured to **Enable XR support**. This generates an exclusive-mode application (fullscreen), which is required to get access to MRC, for security and privacy reasons. Otherwise Unity by default generates a shared application (slate app), and MRC access will be denied by the OS.
+
+### Video capture properties
+
+The video capture properties are related to the video capture itself, that is how the component access the webcam and captures video frames from it, independently of WebRTC.
+
+- The **Start capture when enabled** checkbox, which corresponds to the [`MediaSender.AutoStartOnEnabled`](xref:Microsoft.MixedReality.WebRTC.Unity.MediaSender.AutoStartOnEnabled) property, instructs the component to open the video capture device (webcam) automatically as soon as possible, that is once both:
+  - the component is enabled, that is [`MonoBehaviour.OnEnable()`](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnEnable.html) executed; and
+  - the peer connection is initialized, that is the [`PeerConnection.Initialized`](xref:Microsoft.MixedReality.WebRTC.PeerConnection.Initialized) event was raised.
+
+  This property is checked by default for developer convenience. If unchecked, then the application has to explicitly call [`MediaSender.StartCaptureAsync()`](xref:Microsoft.MixedReality.WebRTC.Unity.MediaSender.StartCaptureAsync) once the two conditions above are true. This can be useful if the application wants to enforce an explicit user action to start video capture, which is generally best practice anyway privacy-wise. Therefore developers are encouraged to uncheck it, and instead bind an end-user action (_e.g._ button click) to a call to [`MediaSender.StartCaptureAsync()`](xref:Microsoft.MixedReality.WebRTC.Unity.MediaSender.StartCaptureAsync).
```

Comment:
So maybe it should be unchecked by default?

Reply:
This is the usual dilemma of the thing that works out-of-the-box but is not the best for production, vs. the best production setting that forces the user to do some setup out-of-the-box before they can make things work. I prefer the latter, and document how to disable it.

Reply:
If this is meant to be used in production then IMHO the default setting should be what's best for production, especially if privacy is concerned. This way we are removing setup cost from prototypes and demos and adding it to real apps (one hast to virtually always uncheck the box) which doesn't seem that clever to me (also, in this case the "setup" is just ticking a box). 

I don't have strong views about this anyway, if you want to keep it like this is fine, just rephrase the docs to something like "this makes the component work out of the box for developer convenience. In cases where privacy is a concern, you might want to keep this unchecked and use an explicit call to .... instead". At the moment the docs read as "we decided to make this checked it by default, but it's better if you uncheck it" which sounds a bit silly 😄

Reply:
Alright, I will rephrase.